torch/distributed/tensor/parallel/_utils.py

```diff
@@ -58,7 +58,7 @@ def wrapper(*args, **kwargs):  # pyre-ignore[2, 3]
                 raise RuntimeError("device_mesh is not passed nor can be inferred")
         if device_mesh.ndim != 1:
             raise RuntimeError(
-                f"device_mesh has dims {device_mesh.ndim} but expcted to be 1"
+                f"device_mesh has dims {device_mesh.ndim} but expected to be 1"
```

Comment:
This change causing a test to fail since the test, checks for the original (mistyped) error string, https://github.com/pytorch/pytorch/actions/runs/4286168169/jobs/7465780948

You can either remove this change or update the corresponding test file in this PR below:
https://github.com/pytorch/pytorch/blob/f33180fb7f7e0773ecc5ff535cfe5faa14469e8d/test/distributed/tensor/parallel/test_tp_style.py#L35-L44

Reply:
Thank you for pointing this out. I have to update this, too. 

I will also update `test_tp_style.py` in this PR.